dev/ComboBox/ComboBoxHelper.cpp

```diff
@@ -101,9 +101,10 @@ void ComboBoxHelper::UpdateCornerRadius(const winrt::ComboBox& comboBox, bool is
 {
     if (comboBox.IsEditable())
     {
-        // TODO, read default values from theme resource once we have them
-        winrt::CornerRadius popupRadius { 0 };
-        winrt::CornerRadius textBoxRadius{ 0 };
+        auto controlRadiusInspectable = winrt::Application::Current().Resources().Lookup(box_value(L"ControlCornerRadius"));
```

Comment:
Does this handle a local override between App.xaml and the control, or only top level resources?

Reply:
Updated the resource lookup to check local scope first.